etc/docker/kernel-tf-py/Dockerfile

```diff
@@ -1,8 +1,19 @@
-# Ubuntu:Bionic
-# TensorFlow 2.4.0
-ARG BASE_CONTAINER=jupyter/tensorflow-notebook:703d8b2dcb88
-
-FROM $BASE_CONTAINER
+FROM jupyter/scipy-notebook:python-3.9.7
+
+# Fix DL4006
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN if [[ `uname -m` == "aarch64" ]] ; then \
```

Comment:
Why do we need tf-io here?

Reply:
Because tensorflow-aarch64 requires tensorflow_io_gcs_filesystem

Reply:
SGTM.